dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -604,6 +612,8 @@
             <SolidColorBrush x:Key="LayerOnAcrylicFillColorDefaultBrush" Color="{ThemeResource SystemColorWindowColor}" />
             <SolidColorBrush x:Key="LayerOnAccentAcrylicFillColorDefaultBrush" Color="{ThemeResource SystemColorWindowColor}"/>
 
+            <SolidColorBrush x:Key="MicaBaseAltFallbackColorBrush" Color="{StaticResource SystemColorWindowColor}" />
```

Comment:
The pattern is to make this a ThemeResource when pointing to a system HC color resource:

<SolidColorBrush x:Key="MicaBaseAltFallbackColorBrush" Color="{ThemeResource SystemColorWindowColor}" />